0.data_analysis/plate_projection_cp_fs_data/nbconverted/plate_1_2_projections.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Create projections with PCA and UMAP
+
+# ## Imports
+
+# In[ ]:
```

Comment:
these are appearing as non-executed code blocks. Please make sure to add sequentially executed notebooks

Reply:
Ahh, that's good to know. I changed the sequence I am executing code.